backend/app.ts

```diff
@@ -1,40 +1,35 @@
 import express, { NextFunction, Request, Response } from "express";
 import path from "path";
+const multer = require("multer");
+const upload = multer();
 
-import { companyRouter } from "./routes/companyRouter";
+import { employeeRouter } from "./routes/employeeRouters";
 import { DBAuthenticationError } from "./error/500s";
 import { statusCodedError } from "./error/statusCodedError";
-const cors = require('cors')
-
+const cors = require("cors");
 
 // -------------------firing express app
 const app = express();
+
+//body paramter enable
+app.use(upload.array());
+
 app.use(express.json());
 app.use(cors());
 app.use(express.urlencoded({ extended: false }));
 app.use(express.static(path.join(__dirname, "client/build")));
 
 // -------------------routes
-app.use("/company", companyRouter);
+// app.use("/company", companyRouter);
```

Comment:
needs explanation

Reply:
Then delete, don't comment. Git keeps our history, in general it's fine to delete things